skopt/gp_opt.py

```diff
@@ -194,12 +194,14 @@ def gp_minimize(func, dimensions, base_estimator=None, alpha=10e-10,
 
     # Default GP
     if base_estimator is None:
+        cov_amplitude = ConstantKernel(1.0, (0.01, 1000.0))
+        matern = Matern(length_scale=np.ones(space.transformed_n_dims),
+                        length_scale_bounds=[(0.01, 100)] * space.transformed_n_dims,
+                        nu=2.5)
+        noise = WhiteKernel()
         base_estimator = GaussianProcessRegressor(
-            kernel=(ConstantKernel(1.0, (0.01, 1000.0)) *
-                    Matern(length_scale=np.ones(space.transformed_n_dims),
-                           length_scale_bounds=[(0.01, 100)] * space.transformed_n_dims,
-                           nu=2.5)),
-            normalize_y=True, alpha=alpha, random_state=random_state)
+            kernel=cov_amplitude * matern + noise,
+            normalize_y=True, random_state=random_state)
```

Comment:
By default `alpha` in `GaussianProcessRegressor` is positive, which means noise can be controlled in two ways. Shall fix `alpha=0.0` instead here, since we have the `noise` component?